.eslintrc

```diff
@@ -3,17 +3,22 @@ plugins:
   - sorting
 extends: plugin:fxa/client
 
+# Required for `import( )` webpack feature
+parser: babel-eslint
+
 rules:
   brace-style: 2
   handle-callback-err: 0
+  indent: 0 # TODO: Write a follow up patch to enable this rule.
   key-spacing: [2, {"beforeColon": false, "afterColon": true}]
   keyword-spacing: 2
   linebreak-style: [2, "unix"]
   max-len: [2, 160]
   new-cap: [2, {"newIsCap": true, "capIsNew": false}]
   no-console: 0
   no-multi-str: 2
+  no-useless-escape: 0
   sorting/sort-object-props: [ 2, { "ignoreCase": true, "ignoreMethods": true, "ignoreObjectsWithMethods": true, "ignorePrivate": true } ]
   space-infix-ops: 2
   space-unary-ops: [2, {"nonwords": false, "overrides": {"~": true, "!": true}}]
-  strict: [2, "never"]
+  strict: 0
```

Comment:
Does webpack insert a `use strict` for us somewhere or is it left to individual modules to decide? I notice it's been removed in some of the diffs below and left alone in others. If webpack isn't doing it for us we should enforce that it's set everywhere using the linter.

Reply:
`babel-loader` automatically adds `use strict`: 

```

/***/ 346:
/***/ (function(module, exports, __webpack_require__) {

"use strict";


var _startupStyles = __webpack_require__(315);

var _startupStyles2 = _interopRequireDefault(_startupStyles);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var startupStyles = new _startupStyles2.default({
  window: window
}); /* This Source Code Form is subject to the terms of the Mozilla Public
     * License, v. 2.0. If a copy of the MPL was not distributed with this
     * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

startupStyles.initialize();
```

Reply:
ref https://github.com/babel/babel-loader#note-transform-runtime--custom-polyfills-eg-promise-library